.github/workflows/ci.yaml

```
@@ -20,14 +20,13 @@ jobs:
             python-version: ["3.11"]
     runs-on: ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - name: Install Poetry
         run: pip3 install poetry==1.5.1
```

```suggestion
        run: pipx install poetry
```

And what about something like that instead?

---

Fails on MacOS at the moment. Trying.